src/prefect/runner/storage.py

```diff
@@ -582,8 +655,12 @@ def create_storage_from_url(
     parsed_url = urlparse(url)
     if parsed_url.scheme == "git" or parsed_url.path.endswith(".git"):
         return GitRepository(url=url, pull_interval=pull_interval)
-    else:
+    elif parsed_url.scheme in fsspec.available_protocols():
```

Comment:
I'm splitting the path that gets passed in now on these characters "://" when the scheme is local or file but I'm not sure if there's maybe a better way to handle that though since I think that can get kinda tricky when you start including drive letters into the mix for windows paths.